frontend/src/scenes/web-analytics/tabs/marketing-analytics/frontend/components/settings/ExternalDataSourceConfiguration.tsx

```diff
@@ -0,0 +1,97 @@
+import { LemonButton, LemonSwitch, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { router } from 'kea-router'
+import { LemonTable } from 'lib/lemon-ui/LemonTable'
+import { DataWarehouseSourceIcon } from 'scenes/data-warehouse/settings/DataWarehouseSourceIcon'
+import { urls } from 'scenes/urls'
+
+import { ExternalDataSource, PipelineNodeTab, PipelineStage } from '~/types'
+
+import { marketingAnalyticsSettingsLogic } from '../../logic/marketingAnalyticsSettingsLogic'
+
+const VALID_REVENUE_SOURCES: ExternalDataSource['source_type'][] = ['BigQuery']
```

Comment:
Make this more reasonable for this component

```suggestion
const VALID_MARKETING_SOURCES: ExternalDataSource['source_type'][] = ['BigQuery']
```